src/content/advanced/iam-roles-for-service-accounts/index.md

```diff
@@ -0,0 +1,105 @@
+---
+linkTitle: IAM Roles for Service Accounts
+title: AWS IAM Roles for Service Accounts 
+description: This article describes how to use a new feature that allows binding of specific AWS IAM roles to a service account of a pod.
+weight: 60
+menu:
+  main:
+    parent: advanced
+user_questions:
+ -  How can I use IAM Roles for Service Accounts?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+last_review_date: 2022-03-04
+---
+
+# IAM Roles for Service Accounts (IRSA)
+
+{{< platform_support_table aws="alpha=v17.1.0" >}}
+
+You can associate an IAM role with a Kubernetes service account. This service account can then provide AWS permissions to the containers in any pod that uses that service account. With this feature, you no longer need to provide extended permissions to the GiantSwarm node IAM role so that pods on that node can call AWS APIs.
```

Comment:
```suggestion
You can associate an IAM role with a Kubernetes service account. This service account can then provide AWS permissions to the containers in any pod that uses that service account. With this feature, you no longer need to provide extended permissions to the Giant Swarm node IAM role so that pods on that node can call AWS APIs.
```